src/marketplace/find-and-install/index.md

```diff
@@ -9,24 +9,40 @@ tags:
   - 'abtnode'
 ---
 
-Your ABT Node Marketplace is your digital catalog of Blocklets from ArcBlock, ecosystem partners and software vendors that make it easy to find, install, manage and one-click deploy Blocklets on your ABT Node.
 
-![](./images/abtnode-overview-2-en.png)
+Your ABT Node Marketplace is your digital catalog of Blocklets from ArcBlock, ecosystem partners, and software vendors that make it easy to find, install, manage and one-click deploy Blocklets on your ABT Node.
+
+![](./images/abtnode-overview.png)
 
 ## Find a Blocklet
 
-To get started, simply go to the left-hand action bar and click on **marketplace**.  Inside of the marketplace, you can relist your catalogue by popularity, or title, with new filters coming soon.
+To get started, go to the left-hand action bar and click on **marketplace**. It will display a catalog of Blocklets created by the ABT Node community. Blocklets are shown with details like the latest release version, release date, install count, and project tags. You can sort the catalog by popularity or by title.
 
 ![](./images/popularity.png)
 
+A *search* box is also provided to filter Blocklets based on their name and description.
+
+![](./images/search-blocklet.png)
+
 ## Install a Blocklet
 
-Installing and working with Blocklets is easy. When you have a Blocklet you want to add to your ABT Node, simply click on **Install Latest** and the Blocklet will automatically install for you.
+Installing and working with Blocklets is easy. When you have a Blocklet you want to add to your ABT Node, click on **Install**, and the Blocklet will automatically install for you.
+
+![](./images/install-blocklet.png)
 
-![](./images/install.png)
+Once the installation is complete, the Install button will change to the *Open* button.
 
-Once installed, be sure to start the Blocklet to begin using it.
+![](./images/open-blocklet-ops.png)
+
+The installation process does not start the Blocklet. You have to access the [Blocklet Management console](/en/blocklet) for starting / stopping / configuring the Blocklet
 
 ![](./images/start.png)
 
-_Note, some blocklets may require additional configuration in order to work correctly so be sure to check your configuration details if additional steps are required.
+_Note, some Blocklets may require additional configuration to work correctly, so be sure to check your configuration details if additional steps are required.
+
+## Register Blocklet Repository
+
+By default, ABT Node marketplace is configured to use Blocklets from the ABT Node and Dev ABT Node repositories. These repositories provide an extensive catalog of community-created Blocklets.
```

Comment:
ABT Node has 2 built-in blocklet registries: the production registry which provide hosting for production ready blocklets for end consumer, and the dev registry provides a lot of sample and demo blocklets for developers to know the posibility of ArcBlock platform.

Reply:
fixed